native/cocos/2d/renderer/UIMeshBuffer.cpp

```diff
@@ -40,9 +59,24 @@ void UIMeshBuffer::setIData(uint16_t* iData) {
     _iData = iData;
 }
 
-void UIMeshBuffer::initialize(gfx::Device* /*device*/, ccstd::vector<gfx::Attribute*>&& attrs, uint32_t /*vFloatCount*/, uint32_t /*iCount*/) {
-    if (attrs.size() == 4) {
-        _attributes.push_back(gfx::Attribute{gfx::ATTR_NAME_COLOR2, gfx::Format::RGBA32F});
+void UIMeshBuffer::initialize(gfx::Device* /*device*/,Vertex2DFormat vfm, uint32_t /*vFloatCount*/, uint32_t /*iCount*/) {
```

Comment:
Need to figure out why can not pass array

Reply:
in c++ get a item value with null vector. i think jsb not convert class. 
i have no time to find root cause, so use a simple way to avoid this problem